src/Monomer/Graphics/FontManager.hs

```diff
@@ -101,11 +103,15 @@ newManager ctx = FontManager {..} where
       }
 
 loadFont :: FMContext -> [Text] -> FontDef -> IO [Text]
-loadFont ctx fonts (FontDef name path) = do
-  res <- fmCreateFont ctx name path
+loadFont ctx fonts fontDef = do
+  res <- createFont fontDef
   if res >= 0
-    then return $ path : fonts
+    then return $ name : fonts
```

Comment:
Since the values inside the resulting `IO [Text]` weren't used and the fonts loaded from memory don't have a path identifying them, this was changed to return a list of font names instead.

Reply:
Makes sense! 👍